src/components/tooltip/styles.ts

```diff
@@ -0,0 +1,132 @@
+import { styled } from '@linaria/react'
+
+export const ElToolTipContainer = styled.div`
+  position: relative;
+  display: inline-block;
+  overflow: visible;
+`
+
+export const ElToolTipChild = styled.div`
+  width: max-content;
+  background: var(--fill-default-darkest, #222b33);
+  color: var(--text-white);
+  padding: var(--spacing-2) var(--spacing-3) var(--spacing-2) var(--spacing-3);
+  border-radius: var(--corner-default);
+  font-family: var(--font-family);
+  font-size: var(--font-size-xs);
+  font-weight: 400; // To do : Need to replace with variable
+  line-height: var(--line-height-xs);
+  letter-spacing: var(--letter-spacing-xs);
+  text-align: left;
+  text-underline-position: from-font;
+  text-decoration-skip-ink: none;
```

Comment:
**question:** where are these underline and decoration properties coming from? Figma?

Reply:
Yes, this is coming from Figma. I'll double-check with Andrei to find a solution for non-required CSS to be in a separate section if not explicitly required for elements.